include/avif/avif.h

```diff
@@ -101,9 +101,9 @@ typedef int avifBool;
 #define AVIF_DEFAULT_IMAGE_COUNT_LIMIT (12 * 3600 * 60)
 
 #define AVIF_QUALITY_DEFAULT -1
-#define AVIF_QUALITY_LOSSLESS 100
 #define AVIF_QUALITY_WORST 0
 #define AVIF_QUALITY_BEST 100
+#define AVIF_QUALITY_LOSSLESS 100
```

Comment:
Note line 108 and 109 are in the opposite order. This group makes sense as it goes worst->best->lossless

Reply:
Yes. I considered this and decided to keep worst and best next to each other, like line 109 and 110.